DATA/production/calib/tpc-laser-aggregator.sh

```diff
@@ -20,7 +20,7 @@ publish_after=440
 min_tracks=0
 num_lanes=36
 
-
+REMAP="--condition-remap \"file:///home/wiechula/processData/inputFilesTracking/triggeredLaser/=GLO/Config/GRPECS;file:///home/wiechula/processData/inputFilesTracking/triggeredLaser/=GLO/Config/GRPMagField;file:///home/wiechula/processData/inputFilesTracking/triggeredLaser=TPC/Calib/LaserTracks\" "
```

Comment:
Just for the record: you don't need to repeat the path if it is the same, could do simply:
```
REMAP="--condition-remap file:///home/wiechula/processData/inputFilesTracking/triggeredLaser/=GLO/Config/GRPECS,GLO/Config/GRPMagField,TPC/Calib/LaserTracks"
```